src/test/java/net/minecraftforge/test/GuiTabsTest.java

```diff
@@ -95,6 +99,17 @@ public void onTabClicked(GuiContainer guiContainer)
                     GuiTabsTest.INSTANCE.sendToServer(new TabMessageHandler.TabMessage(0));
                 }
             }.addTo(GuiInventory.class);
+            ItemStack is = new ItemStack(Items.APPLE);
+            is.addEnchantment(new net.minecraft.enchantment.EnchantmentProtection(Enchantment.Rarity.COMMON, EnchantmentProtection.Type.FALL), 1);
```

Comment:
Just use Enchantments.PROTECTION?
You are instantiating a new `Enchantment` object (like `Item`, `Block`, or `Potion`) without registering it.